src/__tests__/Header.spec.tsx

```diff
@@ -13,7 +13,32 @@ export const headerLinkOptions = [
     url: 'http://wallet.lykke.com'
   },
   {
-    title: MenuItem.Profile,
+    title: MenuItem.Settings,
+    url: '#'
+  }
+];
+
+export const secondaryMenuItems = [
+  {
+    iconName: 'lykke-streams',
+    title: MenuItem.LykkeStreams,
+    url: '#'
```

Comment:
I see that iconName is not required, why can't we do the same with url?

Reply:
Because it is required in all cases and '#' is just a mock